pkg/roachpb/metadata.proto

```diff
@@ -265,12 +265,6 @@ message RangeDescriptor {
   // replica applying the merge is.
   optional int64 generation = 6 [(gogoproto.nullable) = false, (gogoproto.casttype) = "RangeGeneration"];
 
-  // This field is not used any more, but we need to maintain it in 20.2 because
-  // 20.1 nodes need descriptors to round-trip through 20.2 nodes and compare
-  // Equal() when they come back. 20.2 nodes know to ignore this field when
-  // comparing, so the field can be removed in 21.1.
```

Comment:
This change will materialize in 22.1+, correct? (which is past 22.1, so we are good?)

Reply:
Yep, this will be in the upcoming 22.2 release, as there's no reason to backport this to previous releases.